src/assets/text/helloworld.wdl

```diff
@@ -0,0 +1,21 @@
+version 1.0
+
+task hello {
+  input {
+    String name
+  }
+
+  command {
+    echo 'hello ${name}!'
+  }
+  output {
+    File response = stdout()
+  }
+  runtime {
+    docker: 'ubuntu:latest'
```

Comment:
Pinned image
and metadata https://docs.dockstore.org/en/develop/advanced-topics/best-practices/wdl-best-practices.html?highlight=metadata#authorship-metadata

Reply:
Again, might be better to use a modified version of our bamstats example since it's more complete.

Reply:
Whatever works, just want to show a "best practices" example